appveyor_after_build.ps1

```diff
@@ -0,0 +1,300 @@
+. .\appveyor_helpers.ps1
+
+####################### Local run (Debug) configuration ###################################
+
+
+# Set this to true to be able to run outside of appveyor - just configure the section below
+$LOCAL_DEBUG_RUN = $false
+
+
+# Set these input variable that would normally come from AppVeyor yml
+if ($LOCAL_DEBUG_RUN)
+{    
```

Comment:
The 3 variables below will come from Appveyor itself. I've added a mechanism that allows the scripts to run locally. This is great for debugging but can also be used "standalone" to provision a SonarQube server in minutes (~4 minutes to be exact) - without user intervention.